test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test.csproj

```diff
@@ -30,6 +30,30 @@
     <PackageReference Include="Moq" Version="$(MoqVersion)" />
     <PackageReference Include="xunit" Version="$(XunitVersion)" />
     <PackageReference Include="xunit.runner.visualstudio" Version="$(XunitVersion)" />
+
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="2.0.0-*" />
+
+    <!-- We need this to compile but don't want it referenced by the project.
+      It is included dynamically in CodeGeneration tests. -->
+    <ProjectReference Include="..\Microsoft.AspNetCore.Razor.Test.MvcShim\Microsoft.AspNetCore.Razor.Test.MvcShim.csproj">
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+      <OutputItemType>Content</OutputItemType>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Targets>Build</Targets>
+    </ProjectReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- For some reason when the TargetFramework should be 'netcoreapp2.0' we're getting ''. For now we'll hardcode it. -->
```

Comment:
O_o, should definitely figure out why. I feel that this would be a much larger issue if this was true across the board.